PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/OriginalGraphTest.java

```diff
@@ -0,0 +1,49 @@
+package nl.tudelft.pl2016gr2.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class OriginalGraphTest extends GraphTest {
+  
+  private OriginalGraph curGra;
+
+  @Override
+  public GraphInterface getInstance() {
+    OriginalGraph gra = new OriginalGraph();
```

Comment:
This is not a problem because gra in the parent class is set using getInstance(). I gave it the same name so it would be clearer that they have to do with the same variable.